packages/base/src/3dview/helpers.ts

```diff
@@ -275,8 +345,14 @@ export function buildShape(options: {
   boundingBox.visible = false;
```

Comment:
We should now create the bounding box from the `meshGroup` so that it has the proper rotation: `const bbox = new THREE.Box3().setFromObject(mainMesh);` -> `const bbox = new THREE.Box3().setFromObject(meshGroup);`

Reply:
Done, Thanks :)